django_vite/templatetags/django_vite.py

```diff
@@ -53,6 +60,10 @@
     settings, "DJANGO_VITE_LEGACY_POLYFILLS_MOTIF", "legacy-polyfills"
 )
 
+DJANGO_VITE_STATIC_URL = path_join(
+    settings.STATIC_URL, DJANGO_VITE_STATIC_URL_PREFIX, ""
```

Comment:
I don't understand the third parameter (empty string) here.

Reply:
It adds a leading slash to the path if the `DJANGO_VITE_STATIC_URL_PREFIX` doesn't contains one.

It is mandatory that `DJANGO_VITE_STATIC_URL` has a leading slash in order to be used with `urljoin`.
If there is no leading slash the last path would be omitted.

e.g.
```python
>>> from urllib.parse import urljoin

# bad
>>> urljoin("/static/bundler", "main.js")
'/static/main.js'

# good
>>> urljoin("/static/bundler/", "main.js")
'/static/bundler/main.js'

```

Reply:
Ok I see thanks !